### DISCUSSION TITLE
Latex Equation

### DISCUSSION THREAD

**Discussion by xiyuzhai** [emilk/egui#4095] [General]
Is there any plan for equation displaying in egui?

**Comment by xiyuzhai** [comment]
It would be great if there could be something like `Shape::Equation` or `Shape::Tex`. I know there could be too much to ask now, but I'm experimenting on my own by rendering things directly into `Shape::Mesh`.

**Reply by xiyuzhai** [reply]
I would like to image it as a generalization of Galley. Mixing all kinds of stuffs into a paragraph. True rich text editing.

**Reply by nil-is-lin** [reply]
Any good solutions? I'm looking forward to this feature as well. it should be a good point for the latex symbol rendering, as the typst works well.

**Reply by xiyuzhai** [reply]
I am going to implement it as a different kind of galley. I believe under egui framework equation visualization can be made extremely efficient.
